lib/storage-client.js

```diff
@@ -243,6 +243,8 @@ class ChromedriverStorageClient {
       await fs.mv(chromedriverPath, dst, {
         mkdirp: true
       });
+      await fs.chmod(dst, 0o755);
```

Comment:
It is only necessary to move this out of the try/finally block

Reply:
It even makes sense to move permissions change out of this method, since it has nothing to do with unzipping and put it into retrieveDriver method, right next to `this.unzipDriver(archivePath, targetPath)` call

Reply:
Thank you